Makefile

```diff
@@ -1,5 +1,6 @@
 transifex_utils = ./node_modules/.bin/transifex-utils.js
-intl_imports = ./node_modules/frontend-platform-shim/node_modules/.bin/intl-imports.js
+intl_imports = ./node_modules/frontend-platform/node_modules/.bin/intl-imports.js
```

Comment:
@OmarIthawi thanks, I've fixed it, but I got this error:

```
Translations pulled successfully!
./node_modules/.bin/intl-imports.js paragon frontend-app-admin-portal
intl-imports.js: Not creating 'frontend-app-admin-portal/index.js' because no .json translation files were found.
intl-imports.js: Skipped import due to missing 'frontend-app-admin-portal/index.js' likely due to empty translations.
```

as I noticed all translation files were downloaded to `src/i18n/messages/paragon`, then I set `$(intl_imports) paragon frontend-app-admin-portal` to `$(intl_imports) paragon`  and the issue got fixed, please check the PR again and test result.